lib/beaker/host_prebuilt_steps.rb

```diff
@@ -505,10 +522,13 @@ def set_env host, opts
         host.exec(Command.new("touch #{host[:ssh_env_file]}"))
 
         #add the constructed env vars to this host
-        host.add_env_var('RUBYLIB', '$RUBYLIB')
-        host.add_env_var('PATH', '$PATH')
-        env.each_pair do |var, value|
-          host.add_env_var(var, value)
+
+        if host['platform'] !~ /freebsd/
```

Comment:
@anodelman No worries, here's the ones I was using:

```yaml
HOSTS:
  freebsd-10-x64:
    roles:
      - master
    platform: freebsd-10-x64
    box : freebsd-10-x64-vbox
    box_url : https://vagrantcloud.com/chef/boxes/freebsd-10.0/versions/1.0.0/providers/virtualbox.box
    hypervisor : vagrant
CONFIG:
  type: git
HOSTS:
  freebsd-9-x64:
    roles:
      - master
    platform: freebsd-9-x64
    box : freebsd-9-x64-vbox
    box_url : https://vagrantcloud.com/chef/boxes/freebsd-9.2/versions/1.0.0/providers/virtualbox.box
    hypervisor : vagrant
CONFIG:
  type: git
```

You can also test this with a real life example here:

https://github.com/petems/puppetlabs-inifile/tree/freebsd_beaker_tests